ops/model.py

```diff
@@ -197,11 +199,12 @@ def __getitem__(self, key):
 class RelationMapping(Mapping):
     """Map of relation names to lists of Relation instances."""
 
-    def __init__(self, relation_names, our_unit, backend, cache):
+    def __init__(self, relations_meta, our_unit, backend, cache):
+        self._relations_meta = relations_meta
         self._our_unit = our_unit
         self._backend = backend
         self._cache = cache
-        self._data = {relation_name: None for relation_name in relation_names}
+        self._data = {relation_name: None for relation_name in list(relations_meta)}
```

Comment:
Why transforming relations_meta into a list before iterating over it?

Reply:
Agreed, no reason to do this.